### PR TITLE
app code

### PR DESCRIPTION
**Pull request by rajeshgheware**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+@app.route('/')
+def get_env():
+
+    return list_env_vars()
+
+@app.route('/env')
+def list_env_vars():
+    env_vars = os.environ
+    return '\n'.join([f'{key}: {value}' for key, value in env_vars.items()])
```
Comment (brainupgrade-in) [inline app.py:16]: Add description for this method

File: app.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                'longitude': first_result.get('longitude'),
+                'temperature (C)': get_current_temperature(first_result.get('latitude'), first_result.get('longitude'))
+            }
+        else:
+            return None
+    except requests.RequestException as e:
+        print(f"An error occurred while fetching location details: {e}")
+        return None
+
+@app.route('/temperature/<latitude>/<longitude>')
```
Comment (brainupgrade-in) [inline app.py:56]: what this method does?

**Review by brainupgrade-in — CHANGES_REQUESTED** [review]
respond to the comments please

File: app.py (hunk 2 above)
Comment (rajeshgheware) [reply app.py:56]: This obtains temperature for the given city

File: app.py (hunk 1 above)
Comment (rajeshgheware) [reply app.py:17]: done

**Review by brainupgrade-in — APPROVED** [review]

**Pull request merged**